app/src/test/java/com/example/concordia_campus_guide/ModelTests/CoordinatesTest.java

```diff
@@ -28,4 +29,10 @@ public void getSetLongitude() {
         assertEquals(expected, coordinates.getLongitude());
     }
 
+    @Test
+    public void getLatLngTest() {
+        LatLng latLng = new LatLng(45.49702057370776, -73.57907921075821);
```

Comment:
What is the purpose of this?

Reply:
To test the getter and increase coverage? :P

Reply:
haha I see that now.